### PR TITLE
Build Version 1.6.38

### PR DESCRIPTION
**Pull request by akshayurankar48**
### Description
- Improvement: - Improved code quality syntax and security checks for better coding standards and practices.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by vijetaR — APPROVED** [review]

File: inc/widgets-manager/widgets/class-search-button.php [hunk 1]
```diff
@@ -356,13 +363,11 @@ protected function register_search_style_controls() {
 				],
 				'condition'  => [
 					'border_style!' => 'none',
+					'layout!'       => 'icon',
```
Comment (sushmak02) [inline inc/widgets-manager/widgets/class-search-button.php:366]: confirm @akshayurankar48

File: inc/widgets-manager/widgets/class-search-button.php [hunk 2]
```diff
@@ -330,14 +339,12 @@ protected function register_search_style_controls() {
 				],
 				'condition' => [
 					'border_style!' => 'none',
+					'layout!'       => 'icon',
```
Comment (sushmak02) [inline inc/widgets-manager/widgets/class-search-button.php:342]: confirm @akshayurankar48

File: inc/widgets-manager/widgets/class-search-button.php (hunk 1 above)
Comment (akshayurankar48) [reply inc/widgets-manager/widgets/class-search-button.php:366]: tested this , working fine @sushmak02

File: inc/widgets-manager/widgets/class-search-button.php (hunk 2 above)
Comment (akshayurankar48) [reply inc/widgets-manager/widgets/class-search-button.php:342]: tested this , working fine @sushmak02

**Review by sushmak02 — APPROVED** [review]

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**